addons/l10n_sa_edi/views/report_invoice.xml

```diff
@@ -6,31 +6,27 @@
 
             <!--    Add Currency Exchange rate if different currency than SAR    -->
             <xpath expr="//div[hasclass('clearfix')]" position="after">
-                <div t-if="o.company_id.country_id.code == 'SA' and o.currency_id != o.company_id.currency_id"
-                     id="sar_amounts" class="row clearfix ml-auto my-3 text-nowrap table">
+                <table t-if="o.company_id.country_id.code == 'SA' and o.currency_id != o.company_id.currency_id"
```

Comment:
@alny-odoo What is the reason for changing div into table?  xpaths will break on it, so I hope it has some value.

Reply:
Normally table is the more logical one, but as we are changing the existing in stable, it would be good to know why exactly we need the change.